Assets/MRTK/Tests/TestUtilities/PlayModeTestUtilities.cs

```diff
@@ -220,7 +220,10 @@ public static void EnsureInputModule()
                 {
                     CameraCache.Main.gameObject.AddComponent<MixedRealityInputModule>();
                 }
+
+#if !UNITY_2021_1_OR_NEWER
                 inputModule.forceModuleActive = true;
```

Comment:
I am not sure if this has some potential impact to testing in Unity 2021...

Reply:
According to the obsolete message

> 'forceModuleActive has been deprecated. There is no need to force the module awake as StandaloneInputModule works for all platforms'

Which makes it sound like it should now be on everywhere by default